### PR TITLE
Overhauled the BQ readme

### PR DESCRIPTION
**Pull request by chochem**
It is now also pinned in the quest-dev channel

### CODE REVIEW

File: config/betterquesting/Readme.md [hunk 1]
```diff
@@ -1,106 +1,114 @@
...
+3. Before you being making changes you want to click `Fetch origin` to get the latest updates and then start a new branch (`Branch`, then `New Branch`). If you followed 2. that will automatically be based on the GTNewHorizons/GT-New-Horizons-Modpack master branch.
+4. Start GTNH. (you need the full game, not a development environment)
+5. Replace the DefaultQuests folder in your config/betterquesting game files with the one from your github folder.
+6. Then do `/bq_admin default load` in the game.
+7. Double check your questbook is indeed up to date with the latest changes on GTNewHorizons/GT-New-Horizons-Modpack.
+8. Use `/bq_admin edit` to unlock edit mode. Now you can actually start editing quests!
+9. While doing changes, make regular commits to your branch. To do so start by using `/bq_admin default save`. LOOK REALLY CLOSELY AT THE COMMAND AND MAKE SURE IT SAYS *SAVE*. LIKE SERIOUSLY STARE AT IT.
+10. Then replace the github DefaultQuests folder with the one from your game.
```
Comment (D-Cysteine) [inline config/betterquesting/Readme.md:55]: Maybe same here: reminder to delete the folder first, before copying

File: config/betterquesting/Readme.md [hunk 2]
```diff
@@ -1,106 +1,114 @@
...
-705. Just drop the translated lang file into `config/txloader/load/betterquesting/lang` to apply the translations. In previous versions of BetterQuesting, it was necessary to load a custom version of the quest book data for translations to work, but this is no longer needed.
+
+1. Open the quest book and click Edit at the bottom.
+2. Click the tab you want to put the quest, then click Designer.
+3. Click New Quest, and place it in the quest graph.
+4. Click Select Icon and choose an appropriate icon for the quest.
+5. To define requirements, select Edit, then Requirements, then search for the requirement quest on the right hand side with the quest ID or the title. Select + to add the quest to the requirements. The eye symbol lets you select if the connection is always shown/always hidden/only shown with hovering over or shift. Ideally you want to show dependencies but avoid always shown if it leads to excessive line crossing.
```
Comment (D-Cysteine) [inline config/betterquesting/Readme.md:72]: Is it worth mentioning that you can right-click quests to copy the quest ID? Not sure how easy that feature is to discover

File: config/betterquesting/Readme.md [hunk 3]
```diff
@@ -1,106 +1,114 @@
...
+9. Form the Edit Quest window, select Tasks to add new tasks.
+10. For Tasks we usually use several optional retrieval and one retrieval. Other options are possible too but should only be used for a good reason.
+11. There are many more things to finetune: you can toggle if NBT data should matter, can delete parts of the NBT data of the required item, can allow for oredict substitutes, etc. For example, in a bee quest you should delete all NBT data except for the species and then make sure ignoreNBT is set to false.
+12. Select Rewards to add rewards. You can add Choice where the user chooses between items on a list, or item where they get all the items. Generally try to reward similar amounts as other quests in that tab.
+13. If you click edit, advanced, Object, Object, you get more advanced properties about the quest. For example you can change the cooldown if it should have one or toggle if the quest is a main quest.
+
+
+# Moving Quests #
+
+1. If you need to move a quest to a new tab, get the quest ID.
```
Comment (D-Cysteine) [inline config/betterquesting/Readme.md:86]: Here too, maybe worth mentioning right-click to copy?

File: config/betterquesting/Readme.md [hunk 4]
```diff
@@ -1,106 +1,114 @@
+# Main Guidelines for Quests Development #
+
+1. Quests should primarily provide useful information to the player and guide them through the pack progression. But feel free to develop some lore on top of that.
+2. Everybody is welcome to contribute to the questbook. But any bigger changes need to be discussed in the quest-dev channel on discord first.
+3. You should only change quests on topics you have expertise on and have actually played around with yourself. The only exception to this are mechanics/items/machines that are brandnew additions, in which case you should talk to the responsible dev or study the PR that added these things. Ideally get that dev to write the quest or at least help you with it.
+
+
+# Some more Guidelines for Quests Development#
+
+1. Secret quests should only see very limited use for surprise bonusses or riddles.
```
Comment (D-Cysteine) [inline config/betterquesting/Readme.md:10]: nit / spelling: "bonusses" --> "bonuses" (I think)

**Review by D-Cysteine — APPROVED** [review]
Thanks for cleaning this up!

If you think my suggestions are good, but don't feel like making the edits in this PR, feel free to ignore them and I can make them in a follow-up PR

File: config/betterquesting/Readme.md [hunk 5]
```diff
@@ -1,106 +1,114 @@
...
+1. If you need to move a quest to a new tab, get the quest ID.
+2. Click the destination tab, and select the Add/Remove quests button.
+3. Search for the quest id on the right side, click + to add it to the quests for that tab.
+4. Go back to the original tab and select the Add/Remove quests button.
+5. Find the quest in the left side, and hit the - to remove it.
+6. Quests can be in more than one tab. Important information quests should get added to the Tips and Tricks tab. Just dont remove it from the original location in that case.
+7. However, for the most part quest should be in just one tab. This is first, to avoid clutter. We already have many quest even without having all twice! And second, certain things just don't work as well if a quest is in multiple tabs, e.g. dependency lines.
+
+
+# Deleting Quests#
```
[the remark this answers is not in the document]
Comment (chochem) [reply config/betterquesting/Readme.md]: removed

File: config/betterquesting/Readme.md [hunk 6]
```diff
@@ -1,106 +1,114 @@
+# Main Guidelines for Quests Development #
+
+1. Quests should primarily provide useful information to the player and guide them through the pack progression. But feel free to develop some lore on top of that.
+2. Everybody is welcome to contribute to the questbook. But any bigger changes need to be discussed in the quest-dev channel on discord first.
+3. You should only change quests on topics you have expertise on and have actually played around with yourself. The only exception to this are mechanics/items/machines that are brandnew additions, in which case you should talk to the responsible dev or study the PR that added these things. Ideally get that dev to write the quest or at least help you with it.
+
+
+# Some more Guidelines for Quests Development#
```
[the remark this answers is not in the document]
Comment (chochem) [reply config/betterquesting/Readme.md]: removed them all

**Comment by chochem** [comment]
thanks for reading through this! I put all suggestions in.

**Pull request merged**